### DISCUSSION TITLE
UI changes.

### DISCUSSION THREAD

**Discussion by ghost** [mhaeuser/Battery-Toolkit#49] [Ideas]
1- Match the app icon to the one in the status bar.
2- Combine the General and Power pages into one to simplify the app.
3- Redesign the icon to something simpler, like this:
![charge_pause](https://github.com/user-attachments/assets/ebfe8139-c40b-4262-a799-cf951ab60cf7)
![charge_pause_2](https://github.com/user-attachments/assets/3b3d5cbf-bf16-4f50-9420-69bd88911bfd)
These are good examples because they're vertical. When I glance at the status bar, I'll know that the battery is paused, not charging.

**Comment by mhaeuser** [comment]
1. It’s common that the status bar has simplified icons compared to the app icon.
2. The “unnecessary complication” separates user-specific settings and global settings. I didn’t highlight this simply because the settings categories neatly divide user-specific and global implicitly. Possibly adding a better explanation caption to autostart would clear this up.
3. The current icon is not stateful, so I won’t make this change. This would be implicitly resolved by showing a combined state with this: https://github.com/mhaeuser/Battery-Toolkit/issues/10

**Reply by ghost** [reply]
You could combine them on one page & separate with a dotted line or something.
Might as well move the status bar controls inside the settings so that when I click on the icon, it only opens the settings.
I love simplicity.

**Reply by mhaeuser** [reply]
Controls in settings definitely is not going to happen, the whole point of the menu bar extra is a quick access. It also mirrors Apple’s own controls, so that would feel like garbage. The dotted line - well, there probably are ways to resolve this nicely, the question is whether there really is a problem. I’m not sure I remember using a nice app where user-specific and global settings were on the same page, but I’ll keep my eyes open how others handle this.

**Reply by ghost** [reply]
You've got a point on the control thing but just remember to keep it simple, we don't need any more features. Thanks.
